fast64_internal/oot/oot_skeleton.py

```diff
@@ -81,13 +81,26 @@ def toC(self):
 				str(self.getNumLimbs()) + " };\n\n" 
 			data.header = "extern SkeletonHeader " + self.name + ";\n"
 
+		index = 0
+		for limb in limbList:
+			name = (self.name + "_" + limb.boneName).upper()
+			assert limb.index == index
+			if limb.index == 0:
+				data.header += "#define " + name + "_POS_LIMB 0\n"
+				data.header += "#define " + name + "_ROT_LIMB 1\n"
+			else:
+				data.header += "#define " + name + "_LIMB " + str(limb.index+1) + "\n"
+			index += 1
+		data.header += "#define " + self.name.upper() + "_NUM_LIMBS " + str(index+1) + "\n"
+
 		limbData.append(data)
 
 		return limbData
 
 class OOTLimb():
-	def __init__(self, skeletonName, index, translation, DL, lodDL):
+	def __init__(self, skeletonName, boneName, index, translation, DL, lodDL):
 		self.skeletonName = skeletonName
+		self.boneName = boneName
```

Comment:
I think `boneName` can contain invalid characters for a C identifier at that point, like a leading digit, spaces...